### ISSUE TITLE
Unable to activate rendering for IRB

### ISSUE THREAD

**Issue by maneyko** [hirber/hirber#6]
I am running a Rails console via IRB but am unable to activate `Hirb` without doing a hacky workaround. In my environment I am using `Pry` so `hirber` is setting `Pry.config.print` in [`Hirb::View.enable_output_method`](https://github.com/hirber/hirber/blob/v0.8.3/lib/hirb/view.rb#L191-L199) and not setting output formatting for `IRB::Irb`. So when I do `MyModel.all` I get the regular Ruby output instead of the `Hirb` table.

This is what I've been using as my workaround.
```ruby
# Hirber conditionally adds support for different consoles via +elsif+ blocks.
# https://github.com/hirber/hirber/blob/v0.8.3/lib/hirb/view.rb#L199
if defined?(Pry)
  TempPry = Pry
  Object.__send__(:remove_const, :Pry)
  Hirb.enable
  Pry = TempPry
  Object.__send__(:remove_const, :TempPry)
end
```

**Comment by marzdrel** [comment]
Thanks for the report. Could you write what steps are required to reproduce the issue in a minimal rails app? Which gems should be included in order to invoke that problem?

**Comment by maneyko** [comment]
I will see if I can put together a small Rails app. I believe all that's required is `pry` and `activerecord`. The issue is that `hirber` sees that `Pry` is defined and does not do the `IRB` setup as a result in [`Hirb::View.enable_output_method`](https://github.com/hirber/hirber/blob/v0.8.3/lib/hirb/view.rb#L191-L199).

Maybe this could be a configuration option where `Ripl` is the default, then perform the fallbacks as necessary if it's not defined?

**Comment by maneyko** [comment]
I've created a minimal Rails app at https://github.com/maneyko/rails_mini_app to reproduce the issue. Here are the exact steps to reproduce it:

1. Clone the example repo:
    ```bash
    $ git clone https://github.com/maneyko/rails_mini_app
    ```
2. Install gems for Ruby 2.7.4:
  a. Using RVM:
    ```bash
    $ rvm install 2.7.4
    $ cd rails_mini_app/
    $ bundle install
    ```
    b. If you're using Ruby 2.7.x globally:
    ```bash
    $ cd rails_mini_app/
    $ bundle install
    ```

3. Run the Rails console and see that `Hirb` is not showing records properly: `$ rails c`
    ```ruby
    MyTestModel.first(2)
     # => [#<MyTestModel id: 1, a: "b26d43d26a37a53d0fc8068c", b: false, c: 9265, d: 0.3975e2, e: 0.9225e4, f: 0.00040733>, #<MyTestModel id: 2, a: "b76fd0fd94d8e3b27527526f", b: true, c: 2868, d: 0.306e1, e: 0.4834e4, f: 0.00048239>]
    ```
4. Run the Rails console using the patch and see that records show properly: `$ ENABLE_HIRB_PATCH=1 rails c`
    ```ruby
    MyTestModel.first(2)
     # +----+--------------------------+-------+------+-------+--------+------------+
     # | id | a                        | b     | c    | d     | e      | f          |
     # +----+--------------------------+-------+------+-------+--------+------------+
     # | 1  | b26d43d26a37a53d0fc8068c | false | 9265 | 39.75 | 9225.0 | 0.00040733 |
     # | 2  | b76fd0fd94d8e3b27527526f | true  | 2868 | 3.06  | 4834.0 | 0.00048239 |
     # +----+--------------------------+-------+------+-------+--------+------------+
     # 2 rows in set
    ```
The patch definition is at https://github.com/maneyko/rails_mini_app/blob/master/.irbrc#L8.

**Comment by marzdrel** [comment]
@maneyko I moved the dependency from `Gemfile` to `.gemspec`. I am not sure if introducing `pry` globally for the whole test suite is the best solution. I am not sure how to do it better aside from removing constants after a specific set of specs. In the end, it seems to work fine. 😃 Thanks for this PR, I've merged it and released 0.8.5. 👍

**Comment by maneyko** [comment]
Right, probably adding it as a development dependency to the gemspec was a better option. Thanks for taking a look and releasing it!
